admin_guide/manage_nodes.adoc

```diff
@@ -105,70 +105,38 @@ No events.
-You can add a node to an existing OpenShift instance using the `oc create`
-command and supplying a definition for a node object. After the newly-created
-node passes the health checks that are performed by the master, pods can be
-scheduled for the node.
-
-.To Add a Node to an Existing OpenShift Instance:
 
-. Create a definition for the new node.
-.. If you have an existing node object created, you can export the existing
-object to use as a starting point for the new node's
-link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-definition[definition]:
-+
-----
-$ oc export node <node_name> > <filename>
-----
-+
-Modify the new file to meet the desired definition for your new node.
+To add nodes to your existing OpenShift cluster, you can run an Ansible playbook
+that handles installing the node components, generating the required
+certificates, and other important steps. See the
+link:../install_config/install/advanced_install.html#adding-nodes-advanced[advanced
+installation] method for instructions on running the playbook directly.
 
-.. If you do not have an existing node object, create a new file from scratch by
-following the
-link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-definition[example
-definition].
-
-. Create the node object using the file you created:
-+
-----
-$ oc create -f <filename>
-----
-+
-At this point, the master begins validating the new node by performing health
-checks.
-
-. Verify that the node was added by checking the output of the following
-command:
-+
-====
-
-----
-$ oc get nodes
-NAME                 LABELS                                        STATUS
-node1.example.com    kubernetes.io/hostname=node1.example.com      Ready
-node2.example.com    kubernetes.io/hostname=node2.example.com      Ready
-node3.example.com    kubernetes.io/hostname=node3.example.com      Ready
-----
-====
-+
-Pods are not scheduled for the node until the node passes the health checks that
-are performed by the master. When the node passes these health checks, it is
-then placed in the `Ready` condition.
+ifdef::openshift-enterprise[]
+Alternatively, if you used the quick installation method, you can
+link:../install_config/install/quick_install.html#adding-nodes-or-reinstalling-quick[re-run
+the installer to add nodes], which performs the same steps.
+endif::[]
 
+[[deleting-nodes]]
 == Deleting Nodes
-When you delete a node with the CLI, although the node object is deleted
-in Kubernetes, the pods that exist on the node itself are not deleted. However,
-the pods cannot be accessed by OpenShift. The behavior around deleting nodes and
-pods with the CLI is under active development.
+
+When you delete a node with the CLI, the node object is deleted in Kubernetes,
+but the pods that exist on the node itself are not deleted. These pods cannot be
+accessed by OpenShift. The behavior around deleting nodes and pods with the CLI
+is under active development.
```

Comment:
Nah but I think this could use some clarification around "These pods cannot be accessed by OpenShift." Bare pods (not backed by rc) are gone, rc backed pods are scheduled elsewhere, local manifest pods on the node would need to be manually deleted (?).